### ISSUE TITLE
Drag volume control

### ISSUE THREAD

**Issue by MichalNemec** [zezo357/flutter_meedu_videoplayer#42]
Drag volume control is shown only on landscape mode.
Can this be shown on on portrait too?

How to disable this drag volume functionality completely?

**Comment by zezo357** [comment]
I didn't implement the option to disable it , I can add it in next release


And It should work in portrait too, so I will check that

**Comment by MichalNemec** [comment]
its not visible only on android (portrait)

**Comment by zezo357** [comment]
can you test on `flutter_meedu_videoplayer: ^4.0.9-dev.1`

disabling volume swipes can be done like this
```
  final _meeduPlayerController = MeeduPlayerController(
    controlsStyle: ControlsStyle.primary,
    enabledControls: EnabledControls(
      volumeSwipes: false
    )
  );
```

**Comment by MichalNemec** [comment]
volumeSwipes works as intended, brightnessSwipes is working.
When both are false, brightness is stuck and ui is shown forever

**Comment by MichalNemec** [comment]
when i have this, then when i press volume button, drag bar is still showing up
```
    enabledControls: const EnabledControls(
      volumeSwipes: false,
      brightnessSwipes: false,
    ),
```

**Comment by zezo357** [comment]
> volumeSwipes works as intended, brightnessSwipes is working.
> When both are false, brightness is stuck and ui is shown forever

Weird, will test again, that's on mobile right?

**Comment by MichalNemec** [comment]
yes, tested on android.

**Comment by zezo357** [comment]
fixed on `4.0.9`
